release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -772,6 +772,15 @@ If you customized the default OpenSSH `/etc/ssh/sshd_config` server configuratio
 
 The {cert-manager-operator} 1.11 is now generally available in {product-title} 4.14 as well as {product-title} 4.13 and {product-title} 4.12.
 
+[discrete]
+[id="ocp-4-14-ovn-k-interconnect"]
+=== OVN-Kuberenetes interconnect
+{product-title} {product-version} introduces OVN-Kubernetes interconnect architecture to {openshift-networking}. This techincal change enables scaling clusters linearly with the number of nodes with a perfomance that is similar to OpenShift SDN.
+
+The new architecture also removes external Transport Layer Security (TLS) over proxy connection requirement from each hosted cluster to management cluster, which fulfills hosted control plane requirements. Users can now place more hosted cluster control planes on the management cluster.
```

Comment:
```suggestion
The new architecture also removes the external Transport Layer Security (TLS) over proxy connection requirement from each hosted cluster to management cluster, which fulfills hosted control plane requirements. Users can now place more hosted cluster control planes on the management cluster.
```

Reply:
we shouldn't say this "which fulfills hosted control plane requirements." its not officially meant to be communicated to customers.. neither should we say "Users can now place more hosted cluster control planes on the management cluster.", users don't do that, isn't it the SREs and SD infra team that does this? -> these points are for internal RH communication, I think you picked these up from TE slide deck, that is not meant for users I think...